### PR TITLE
Get this robot running for the first time

### PR DESCRIPTION
**Pull request by JohnGilb**

### CODE REVIEW

File: CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java [hunk 1]
```diff
@@ -34,13 +35,22 @@
     @Inject
     public PoseSubsystem(WPIFactory factory, PropertyManager propManager) {
         log.info("Creating PoseSubsystem");
-        imu = factory.getGyro();
-        leftDistanceSensor = factory.getAnalogDistanceSensor(1, voltage -> VoltageMaps.placeholder(voltage));
+        imu = factory.getGyro(ImuType.navX);
+        leftDistanceSensor = factory.getAnalogDistanceSensor(1, voltage -> TemporaryVoltageMap.placeholder(voltage));
         leftSensorMountingDistanceInches = propManager.createPersistentProperty("LeftSensorMountingDistanceInches", 16.0);
-        
+        currentHeadingProp = propManager.createEphemeralProperty("CurrentHeading", 0.0);
```
Comment (aschokking) [inline CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java]: How do we want to account for different initial headings? That's always been a challenge.

**Comment by aschokking** [comment]
lgtm;

File: CompetitionTemplate/robot-program-build.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
   <!-- TODO: Do we want this commented out? -->
   <!-- Original commit that made this change said "Something is wrong with deploying the build on my computer, so disabling copying log config" -->
```
Comment (WasabiFan) [inline CompetitionTemplate/robot-program-build.xml]: If the line has been uncommented, we should remove the TODO notice (both lines)

File: DeployJRE/.project [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (WasabiFan) [inline DeployJRE/.project]: I don't see a reason to keep this project; all we need is the ant script, right?

**Comment by WasabiFan** [comment]
Shouldn't the deploy scripts go in the common lib, given that we will use it between years?

File: DeployJRE/.project (hunk 3 above)
Comment (JohnGilb) [inline DeployJRE/.project]: I like having it show up in Eclipse this way, but no strong preference.

**Comment by JohnGilb** [comment]
Yeah, the deploy scripts can go to commonlib, since it needs them anyway

File: DeployJRE/.project (hunk 3 above)
Comment (WasabiFan) [inline DeployJRE/.project]: Not using the project eliminates the need to import an extra project; as I see it, that can be a significant time saver when you are switching between different local clones of the project.

**Pull request merged**